src/middlewared/middlewared/plugins/vm.py

```diff
@@ -927,6 +941,11 @@ def random_mac(self):
 
         `devices` is a list of virtualized hardware to add to the newly created Virtual Machine.
         Failure to attach a device destroys the VM and any resources allocated by the VM devices.
+
+        FreeNAS allows a maximum of 16 guest virtual CPU's. By default, it configures every virtual CPU as a
```

Comment:
We should also avoid mentioning FreeNAS, please ;) . Same will show for TrueNAS.

"Maximum of 16 guest virtual CPUs are allowed" or something

Reply:
You are absolutely correct ( again ) :p Apologies, i have corrected it ; )